src/main/java/se/sundsvall/quotationrequest/service/mapper/HelpdeskMapper.java

```diff
@@ -1,26 +1,26 @@
 package se.sundsvall.quotationrequest.service.mapper;
 
+import static generated.se.sundsvall.seab.lime.Receivedthrough.KeyEnum.FORM;
 import static org.apache.commons.lang3.math.NumberUtils.INTEGER_ZERO;
 import static org.apache.commons.lang3.math.NumberUtils.toInt;
-import static se.sundsvall.generated.clients.limeseab.Receivedthrough.KeyEnum.FORM;
 
 import java.util.Optional;
 
-import se.sundsvall.generated.clients.limeseab.Helpdesk;
-import se.sundsvall.generated.clients.limeseab.Receivedthrough;
+import generated.se.sundsvall.seab.lime.Helpdesk;
+import generated.se.sundsvall.seab.lime.Receivedthrough;
 import se.sundsvall.quotationrequest.api.model.QuotationRequest;
 
 public class HelpdeskMapper {
 
 	static final String DESCRIPTION_FORMAT_TEMPLATE = "%s %s\nTel: %s\nEmail: %s\n\n%s";
-	static final Receivedthrough RECEIVEDTHROUGH = new Receivedthrough()
+	static final generated.se.sundsvall.seab.lime.Receivedthrough RECEIVEDTHROUGH = new Receivedthrough()
```

Comment:
No need to define the whole path as the class has been imported, right?

Reply:
No, this was just left-overs from the refactoring. Cleaned up now!